docs/kf.dev/content/en/docs/developers/troubleshooting/_index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Troubleshooting"
+linkTitle: "Troubleshooting"
+weight: 30
+description: >
+  Learn how to troubleshoot common error messages you may encounter when
+  deploying or running an application with Kf.
+---
+
+## Executable jars
+[executable-jar]: https://docs.spring.io/spring-boot/docs/current/reference/html/deployment-install.html 
+If you encounter the error message `zip: not a valid zip file` while deploying
+your Spring application, you may be attempting to deploy an [executable
+jar][executable-jar]. Executable jars are not supported by Kf's build system.
```

Comment:
please capitalize JAR, and JARs.

Reply:
Hah! I actually converted to lowercase after reading the Spring docs that do it that way (https://docs.spring.io/spring-boot/docs/current/reference/html/deployment-install.html). I'll go back to upper :)

Reply:
Done.